lib/auth/default_current_user_provider.rb

```diff
@@ -385,7 +385,9 @@ def lookup_api_user(api_key_value, request)
         end
 
       if user && can_write?
-        api_key.update_columns(last_used_at: Time.zone.now)
+        Scheduler::Defer.later "Updating Last Used" do
```

Comment:
```suggestion
        Scheduler::Defer.later "Updating api_key last_used" do
```
I think mentioning 'api key' here might be useful for debugging